packages/models/src/lib/category-config.ts

```diff
@@ -0,0 +1,66 @@
+import { z } from 'zod';
+
+/**
+ *
+ * Define Zod schema for the CategoryConfig type
+ *
+ * @example
+ *
+ * // Example data for the CategoryConfig type
+ * const categoryConfigData = {
+ *   slug: 'performance',
+ *   title: 'Performance Metrics',
+ *   description: 'This category includes performance-related metrics.',
+ *   metrics: [
+ *     { ref: 'eslint#max-lines', weight: 0.5 },
+ *     { ref: 'categories:lhci#performance', weight: 0.8 },
```

Comment:
Shouldn't `weight`s only be integers? :thinking: Lighthouse only uses integers, and using decimals is confusing.

Reply:
fixed as well as added int and nonnegative